template/deploy/helm/[[operator]]/templates/deployment.yaml.j2

```diff
@@ -17,8 +17,8 @@ spec:
     metadata:
       annotations:
         internal.stackable.tech/image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
-        {{- with .Values.podAnnotations }}
         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
```

Comment:
Should we add a qualifier to the annotation?

Reply:
It appears there are no strong feelings here. I will leave it as is. It can be changed in future if need be.